kolibri/core/content/apps.py

```diff
@@ -13,3 +17,7 @@ class KolibriContentConfig(AppConfig):
     def ready(self):
         from .signals import reorder_channels_upon_deletion  # noqa: F401
         from .signals import cascade_delete_node  # noqa: F401
+
+        manager = ContentAssignmentManager()
+        manager.on_downloadable_assignment(add_download_requests)
+        manager.on_removable_assignment(add_removal_requests)
```

Comment:
Since these are `classmethods` they should be called on the class definition, not an instance
```suggestion
        ContentAssignmentManager.on_removable_assignment(add_removal_requests)
```